<?xml version="1.0" encoding="UTF-8"?>
<ui version="6.0">
 <class>styleEditor</class>
 <widget class="QDialog" name="styleEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Style Editor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget_editor">
     <property name="tabPosition">
      <enum>QTabWidget::West</enum>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="usesScrollButtons">
      <bool>false</bool>
     </property>
     <property name="movable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="tab_main">
      <attribute name="title">
       <string>Main</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="1">
        <widget class="QLabel" name="label_name">
         <property name="toolTip">
          <string>Required</string>
         </property>
         <property name="text">
          <string>Name</string>
         </property>
         <property name="buddy">
          <cstring>lineEdit_name</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLineEdit" name="lineEdit_name">
         <property name="toolTip">
          <string>Name</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="label_category">
         <property name="toolTip">
          <string>Required</string>
         </property>
         <property name="text">
          <string>Category</string>
         </property>
         <property name="buddy">
          <cstring>lineEdit_category</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="SmartLineEdit" name="lineEdit_category">
         <property name="toolTip">
          <string>Category</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLabel" name="label_categoryNumber">
         <property name="toolTip">
          <string>Required</string>
         </property>
         <property name="text">
          <string>Category number</string>
         </property>
         <property name="buddy">
          <cstring>lineEdit_categoryNumber</cstring>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="SmartLineEdit" name="lineEdit_categoryNumber">
         <property name="toolTip">
          <string>Category number</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLabel" name="label_styleLetter">
         <property name="toolTip">
          <string>Required</string>
         </property>
         <property name="text">
          <string>Style letter</string>
         </property>
         <property name="buddy">
          <cstring>lineEdit_styleLetter</cstring>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="SmartLineEdit" name="lineEdit_styleLetter">
         <property name="toolTip">
          <string>Style letter</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLabel" name="label_styleGuide">
         <property name="toolTip">
          <string>Required</string>
         </property>
         <property name="text">
          <string>Style guide</string>
         </property>
         <property name="buddy">
          <cstring>lineEdit_styleGuide</cstring>
         </property>
        </widget>
       </item>
       <item row="6" column="2">
        <widget class="SmartLineEdit" name="lineEdit_styleGuide">
         <property name="toolTip">
          <string>Style guide</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QLabel" name="label_type">
         <property name="toolTip">
          <string>Required</string>
         </property>
         <property name="text">
          <string>Type</string>
         </property>
        </widget>
       </item>
       <item row="7" column="2">
        <widget class="BtComboBox" name="comboBox_type">
         <property name="toolTip">
          <string>Type of beverage</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_ranges">
      <attribute name="title">
       <string>Ranges</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="1">
        <widget class="QLabel" name="label_min">
         <property name="text">
          <string>Min</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_max">
         <property name="text">
          <string>Max</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="SmartLabel" name="label_og">
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <property name="text">
          <string>OG</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="SmartLineEdit" name="lineEdit_ogMin">
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="SmartLineEdit" name="lineEdit_ogMax">
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="SmartLabel" name="label_fg">
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <property name="text">
          <string>FG</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="SmartLineEdit" name="lineEdit_fgMin">
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="SmartLineEdit" name="lineEdit_fgMax">
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="SmartLabel" name="label_ibu">
         <property name="text">
          <string>IBUs</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="SmartLineEdit" name="lineEdit_ibuMin">
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="SmartLineEdit" name="lineEdit_ibuMax">
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="SmartLabel" name="label_color">
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <property name="text">
          <string>Color (SRM)</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="SmartLineEdit" name="lineEdit_colorMin">
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="SmartLineEdit" name="lineEdit_colorMax">
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="SmartLabel" name="label_carb">
         <property name="text">
          <string>Carb (vols)</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="SmartLineEdit" name="lineEdit_carbMin">
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="SmartLineEdit" name="lineEdit_carbMax">
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_abv">
         <property name="text">
          <string>ABV (pct)</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="SmartLineEdit" name="lineEdit_abvMin">
        </widget>
       </item>
       <item row="6" column="2">
        <widget class="SmartLineEdit" name="lineEdit_abvMax">
        </widget>
       </item>
       <item row="7" column="1">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>

     <widget class="QWidget" name="tab_aroma">
      <attribute name="title">
       <string>Aroma</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_aroma">
       <item>
        <widget class="QTextEdit" name="textEdit_aroma">
        </widget>
       </item>
      </layout>
     </widget>

     <widget class="QWidget" name="tab_appearance">
      <attribute name="title">
       <string>Appearance</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_appearance">
       <item>
        <widget class="QTextEdit" name="textEdit_appearance">
        </widget>
       </item>
      </layout>
     </widget>

     <widget class="QWidget" name="tab_flavor">
      <attribute name="title">
       <string>Flavor</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_flavor">
       <item>
        <widget class="QTextEdit" name="textEdit_flavor">
        </widget>
       </item>
      </layout>
     </widget>

     <widget class="QWidget" name="tab_mouthfeel">
      <attribute name="title">
       <string>Mouthfeel</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_mouthfeel">
       <item>
        <widget class="QTextEdit" name="textEdit_mouthfeel">
        </widget>
       </item>
      </layout>
     </widget>

     <widget class="QWidget" name="tab_overallImpression">
      <attribute name="title">
       <string>Overall Impression</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_overallImpression">
       <item>
        <widget class="QTextEdit" name="textEdit_overallImpression">
        </widget>
       </item>
      </layout>
     </widget>

     <widget class="QWidget" name="tab_ingredients">
      <attribute name="title">
       <string>Ingredients</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_ingredients">
       <item>
        <widget class="QTextEdit" name="textEdit_ingredients">
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_examples">
      <attribute name="title">
       <string>Examples</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_examples">
       <item>
        <widget class="QTextEdit" name="textEdit_examples">
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_notes">
      <attribute name="title">
       <string>Notes</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_notes">
       <item>
        <widget class="QTextEdit" name="textEdit_notes"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_new">
       <property name="toolTip">
        <string>New style</string>
       </property>
       <property name="text">
        <string>New</string>
       </property>
       <property name="icon">
        <iconset resource="../resources.qrc">
         <normaloff>:/images/smallPlus.svg</normaloff>:/images/smallPlus.svg</iconset>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_save">
       <property name="toolTip">
        <string>Save and close</string>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
       <property name="icon">
        <iconset resource="../resources.qrc">
         <normaloff>:/images/filesave.svg</normaloff>:/images/filesave.svg</iconset>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_cancel">
       <property name="toolTip">
        <string>Discard and close</string>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="icon">
        <iconset resource="../resources.qrc">
         <normaloff>:/images/exit.svg</normaloff>:/images/exit.svg</iconset>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BtComboBox</class>
   <extends>QComboBox</extends>
   <header>widgets/BtComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>SmartLabel</class>
   <extends>QLabel</extends>
   <header>widgets/SmartLabel.h</header>
   <slots>
    <signal>changedSystemOfMeasurementOrScale(PreviousScaleInfo)</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>SmartLineEdit</class>
   <extends>QLineEdit</extends>
   <header>widgets/SmartLineEdit.h</header>
   <slots>
    <slot>lineChanged(PreviousScaleInfo)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lineEdit_name</tabstop>
  <tabstop>lineEdit_category</tabstop>
  <tabstop>lineEdit_categoryNumber</tabstop>
  <tabstop>lineEdit_styleLetter</tabstop>
  <tabstop>lineEdit_styleGuide</tabstop>
  <tabstop>comboBox_type</tabstop>
  <tabstop>lineEdit_ogMin</tabstop>
  <tabstop>lineEdit_ogMax</tabstop>
  <tabstop>lineEdit_fgMin</tabstop>
  <tabstop>lineEdit_fgMax</tabstop>
  <tabstop>lineEdit_ibuMin</tabstop>
  <tabstop>lineEdit_ibuMax</tabstop>
  <tabstop>lineEdit_colorMin</tabstop>
  <tabstop>lineEdit_colorMax</tabstop>
  <tabstop>lineEdit_carbMin</tabstop>
  <tabstop>lineEdit_carbMax</tabstop>
  <tabstop>lineEdit_abvMin</tabstop>
  <tabstop>lineEdit_abvMax</tabstop>
  <tabstop>pushButton_new</tabstop>
  <tabstop>pushButton_save</tabstop>
  <tabstop>pushButton_cancel</tabstop>
  <tabstop>textEdit_aroma</tabstop>
  <tabstop>textEdit_appearance</tabstop>
  <tabstop>textEdit_flavor</tabstop>
  <tabstop>textEdit_mouthfeel</tabstop>
  <tabstop>textEdit_overallImpression</tabstop>
  <tabstop>textEdit_ingredients</tabstop>
  <tabstop>textEdit_examples</tabstop>
  <tabstop>textEdit_notes</tabstop>
 </tabstops>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
</ui>
